docs/protocol-spec.md

```diff
@@ -2195,9 +2195,9 @@ as depicted in Sections 4.7.1 and 4.7.2.
  - Client side enrollment and transaction creation is performed entirely by a
    non-validating peer that is trusted not to impersonate the user.
    See, Section 4.7.1 for more information.
- - A minimal set of confidentiality properties where a chain-code is accessible
+ - A minimal set of confidentiality properties where a chaincode is accessible
    by any entity that is member of the system, i.e., validators and users who
-   have registered to our membership services, and not accessible by any-one else.
+   have registered to our membership services, and not accessible by anyone else.
```

Comment:
have registered **through Hyperledger Fabric's Membership Services** and not ...
(through or using)